QUICKSTART.md

```diff
@@ -1,58 +1,536 @@
+
+- Python 2.6 or Python 2.7
+
+**Note:** The Control Machine can be one of the managed nodes with access to external network to download binary.
+
+### 2. Install Ansible in the Control Machine
+
+Install Ansible 2.3 or later to your CentOS 7.3 platform: 
 
 ```
-make server TARGET=$GOPATH/bin/tidb-server
+ yum install epel-release
+ yum update
+ yum install ansible
 ```
 
-The default server port is `4000` and can be changed by flag `-P <port>`.
+You can use the `ansible –version` command to see the version information.
+
+For more information, see [Ansible Documentation](http://docs.ansible.com/ansible/intro_installation.html).
+
 
-Run `./tidb-server -h` to see more flag options.
+### 3. Download TiDB-Ansible to the Control Machine
+Download the latest master version of the ZIP package from GitHub [TiDB-Ansible project](https://github.com/pingcap/tidb-ansible) or [click to download]( https://github.com/pingcap/tidb-ansible/Archive/master.zip).
+ 
+ 
+### 4. Allocate machine resources and edit the `inventory.ini` file in the Control Machine
 
-After you started tidb-server, you can use a MySQL client to connect to TiDB.
+The standard Cluster has 6 machines: 
 
+- 2 TiDB instances, the first TiDB instance is used as a monitor 
+- 3 PD instances
+- 3 TiKV instances
+ 
+The cluster topology is as follows:
+
+| Name | Host IP | Services |
+| ---- | ------- | -------- |
+| node1 | 172.16.10.1 | PD1, TiDB1 |
+| node2 | 172.16.10.2 | PD2, TiDB2 |
+| node3 | 172.16.10.3 | PD3 |
+| node4 | 172.16.10.4 | TiKV1 |
+| node5 | 172.16.10.5 | TiKV2 |
+| node6 | 172.16.10.6 | TiKV3 |
+ 
+Edit the `inventory.ini` file:
+ 
 ```
-mysql -h 127.0.0.1 -P 4000 -u root -D test --prompt="tidb> " 
+[tidb_servers]
+172.16.10.1
+172.16.10.2
+ 
+[pd_servers]
+172.16.10.1
+172.16.10.2
+172.16.10.3
+ 
+[tikv_servers]
+172.16.10.4
+172.16.10.5
+172.16.10.6
+ 
+[monitored_servers:children]
+tidb_servers
+tikv_servers
+pd_servers
+ 
+[monitoring_servers]
+172.16.10.1
+ 
+[grafana_servers]
+172.16.10.1
+
+...
+
 ```
+ 
+### 5. Deploy the TiDB cluster
+
+Use the normal user with the sudo privileges to deploy TiDB:
+	
+5.1 Edit the `inventory.ini` file as follows:
+	
+	 ```
+	  ## Connection
+	  # ssh via root:
+	  # ansible_user = root
+	  # ansible_become = true
+	  # ansible_become_user = tidb
+	  
+	  # ssh via normal user
+	  ansible_user = tidb
+	 ```
+5.2 Connect to the network and download the TiDB, TiKV, and PD binaries:
+		
+		ansible-playbook local_prepare.yml
+	                                           
+5.3 Initialize the system environment of the target machines and modify the kernel parameters:
+	
+		ansible-playbook bootstrap.yml -k -K
+	 
+**Note:** 
+	- Add the `-k` (lowercase) parameter if password is needed to connect to the managed node. This applies to other playbooks as well.
+	- Add the `-K`(uppercase) parameter because this playbook needs root privileges.
```

Comment:
- Add the `-K`(uppercase) parameter if sudo needs password, this playbook needs root privileges.